examples/demo_coordinate_system_datasets.ipynb

```diff
@@ -0,0 +1,916 @@
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we create a `GalaxyCluster` object, specifying an *incorrect* coordinate system.  For source galaxies from CosmoDC2, these are in the **euclidean** coordinate system.  We use the implemented kwarg when defining the galaxy cluster object to specify the **celestial** coordinate system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a GalaxyCluster.\n",
+    "cluster = clmm.GalaxyCluster(\"Name of cluster\", cluster_ra, cluster_dec, cluster_z, galaxies, coordinate_system=\"celestial\")\n",
+    "\n",
+    "# Convert elipticities into shears for the members.\n",
+    "cluster.compute_tangential_and_cross_components()\n",
+    "print(cluster.galcat.colnames)\n",
+    "\n",
+    "# Measure profile and add profile table to the cluster.\n",
+    "seps = convert_units(cluster.galcat[\"theta\"], \"radians\", \"Mpc\", cluster.z, cosmo)\n",
+    "\n",
+    "cluster.make_radial_profile(\n",
+    "    bins=da.make_bins(0.1, 3.0, 15, method=\"evenlog10width\"),\n",
+    "    bin_units=\"Mpc\",\n",
+    "    cosmo=cosmo,\n",
+    "    include_empty_bins=False,\n",
+    "    gal_ids_in_bins=True,\n",
+    ")\n",
+    "print(cluster.profile.colnames)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we plot the resulting profile when `clmm` uses assumes a coordinate system inconsistent with the catalogs provided.  You will note that the signal is virtually zero at almost all radii."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(6, 4))\n",
+    "ax = fig.add_axes((0, 0, 1, 1))\n",
+    "errorbar_kwargs = dict(linestyle=\"\", marker=\"o\", markersize=1, elinewidth=0.5, capthick=0.5)\n",
+    "ax.errorbar(\n",
+    "    cluster.profile[\"radius\"],\n",
+    "    cluster.profile[\"gt\"],\n",
+    "    cluster.profile[\"gt_err\"],\n",
+    "    c=\"k\",\n",
+    "    **errorbar_kwargs\n",
+    ")\n",
+    "ax.set_xlabel(\"r [Mpc]\", fontsize=10)\n",
+    "ax.set_ylabel(r\"$g_t$\", fontsize=10)\n",
+    "ax.grid(lw=0.3)\n",
+    "ax.minorticks_on()\n",
+    "ax.grid(which=\"minor\", lw=0.1)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we create a `GalaxyCluster` object, specifying the *correct* coordinate system.  For source galaxies from CosmoDC2, these are in the **euclidean** coordinate system.  We use the implemented kwarg when defining the galaxy cluster object to also specify the **euclidean** coordinate system.  However, with a single galaxy cluster, the signal is not significant enough to clearly see a difference.  We do notice an excess with the correct coordinate system at larger radii."
```

Comment:
Isn't this problem coming from DC2? Because when you try the process with HSC, the signal is pretty visible.

Reply:
Ah, okay - will loop in @ts485 for this discussion while I make some of the minor changes rn.

Reply:
Still need to address this